src/main/java/org/launchcode/techjobs/persistent/models/Entity.java

```diff
@@ -0,0 +1,4 @@
+package org.launchcode.techjobs.persistent.models;
+
+public @interface Entity {
+}
```

Comment:
```suggestion
```
I think this Entity interface might have been created by accident while you added the annotation to one of your classes. Deleting this and importing @Entity from import javax.persistence* will resolve the IllegalArgumentsException you're getting.